mysql-test/suite/rocksdb/t/autoincrement_crash_safe.test

```diff
@@ -0,0 +1,30 @@
+--source include/have_rocksdb.inc
+--source include/have_debug.inc
+
+CREATE TABLE t (i INT PRIMARY KEY AUTO_INCREMENT, b INT) ENGINE=rocksdb;
+
+INSERT INTO t VALUES (NULL,1), (NULL,2), (NULL,3);
+INSERT INTO t VALUES (NULL,4), (NULL,5), (NULL,6);
+SELECT * FROM t;
+DELETE FROM t WHERE i>3;
+SELECT * FROM t;
+SELECT LAST_INSERT_ID();
+
+--echo # crash_during_online_index_creation
+FLUSH LOGS;
+
+--exec echo "restart" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+SET SESSION debug="+d,crash_during_online_index_creation";
+--error 2013
+ALTER TABLE t ADD INDEX kb(b), ALGORITHM=INPLACE;
+
+--enable_reconnect
+--source include/wait_until_connected_again.inc
+
+SET SESSION debug="-d,crash_during_online_index_creation";
```

Comment:
Server is freshly restarted here, no need to remove crash_during_online_index_creation from the session variable